packages/signals/signals/src/core/signals/settings.ts

```diff
@@ -79,58 +93,80 @@ export class SignalGlobalSettings {
      * Add new URLs to the disallow list
      */
     disallowListURLs: (string | undefined)[]
+    /**
+     * Sample rate to determine sending signals
+     */
```

Comment:
Can we make this | undefined / optional, so update can be used atomically (one day?)